scss/mixins/_utilities.scss

```diff
@@ -9,8 +23,6 @@
   }
 
   @each $key, $value in $values {
-    $properties: map-get($utility, property);
```

Comment:
And the second one, here.

Any opinion? What do you think about this?

Reply:
Here we are in a loop and the check will happen for each value. This may result in too many warnings.